src/Application/Models/Asset.js

```diff
@@ -0,0 +1,13 @@
+class Asset {
+  constructor(id, name, baseIncome, price, multiplier, owned, unlocked) {
+    this.id = id;
+    this.name = name;
+    this.baseIncome = baseIncome;
+    this.price = price;
+    this.multiplier = multiplier;
+    this.owned = owned;
+    this.unlocked = unlocked;
+  }
```

Comment:
in order to (if somewhat symbolically, but importantly) pass this class off as an immutable POJO-equivalent class we name all these instance variables `this._id` and the like.  the leading underscore is a pretty trusted convention for "don't touch unless you know probably much more than you do" so these effectively become private fields.  then just make getters that return the private variables, and es6 getters mean it doesn't have to be invoked like a function so that's nice